Makefile

```diff
@@ -81,12 +87,12 @@ psql: .env .docker-build  ## | Open psql cli.
 
 .PHONY: test
 test: .env .docker-build  ## | Run Python unit test suite.
-	${DC} up -d db redis-cache localstack statsd oidcprovider
+	${DC} up --detach ${SERVICES}
```

Comment:
I changed the flag from `-d` to the more descriptive and human-friendly `--detach`.